<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ake-PC" name="com.wymall.test.testcases.system.role.RolePage_001_addRoleSuccess_Test" tests="1" failures="0" timestamp="28 Aug 2018 05:56:26 GMT" time="17.321" errors="1">
  <testcase name="addRoleSuccess" time="17.321" classname="com.wymall.test.testcases.system.role.RolePage_001_addRoleSuccess_Test">
    <error type="java.lang.AssertionError" message="超时!! 10 秒之后还没找到元素 [By.xpath: //*[@id=&quot;app&quot;]//label[contains(text(),&#039;角1色名称&#039;)]/following-sibling::*[1]//input]">
      <![CDATA[java.lang.AssertionError: 超时!! 10 秒之后还没找到元素 [By.xpath: //*[@id="app"]//label[contains(text(),'角1色名称')]/following-sibling::*[1]//input]
	at org.testng.Assert.fail(Assert.java:94)
	at com.wymall.test.utils.SeleniumUtil.waitForElementToLoad(SeleniumUtil.java:279)
	at com.wymall.test.utils.SeleniumUtil.type(SeleniumUtil.java:237)
	at com.wymall.test.utils.actions.impl.SetValue.doAction(SetValue.java:27)
	at com.wymall.test.utils.SuperAction.parseExcel(SuperAction.java:277)
	at com.wymall.test.testcases.system.role.RolePage_001_addRoleSuccess_Test.addRoleSuccess(RolePage_001_addRoleSuccess_Test.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:369)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- addRoleSuccess -->
</testsuite> <!-- com.wymall.test.testcases.system.role.RolePage_001_addRoleSuccess_Test -->
